libraries/AP_Common/Location.cpp

```diff
@@ -194,7 +194,7 @@ bool Location::get_vector_xy_from_origin_NE(Vector2f &vec_ne) const
         return false;
     }
     vec_ne.x = (lat-ekf_origin.lat) * LATLON_TO_CM;
```

Comment:
If you are flying near the poles isn't it hazardous to subtract latitudes as well?

Reply:
no, latitude doesn't wrap, it just hits a limit of 90. We do have issues at the poles which will take a separate PR to fix, but it isn't the same issue as longitude wrap

Reply:
It doesn't wrap but if you fly right over the pole from a latitude of +89 to +89, the velocity shouldn't be zero right?

Reply:
Maybe that's what you meant by other issues.